### PR TITLE
Slim header- redundant styles cleanup

### LINKED ISSUE #8867: Page listing header: clean up styles
Follow-up to #8792. 

- [form-side](https://github.com/wagtail/wagtail/blob/main/client/scss/layouts/_form-side.scss) should move from layouts to components.
- [page explorer index styles](https://github.com/wagtail/wagtail/blob/efaad2593217001c3505c5f1ce7a558a882737a2/client/scss/components/_listing.scss#L483-L529) are now unused and should be removed
- 90% sure [other index styles](https://github.com/wagtail/wagtail/blob/efaad2593217001c3505c5f1ce7a558a882737a2/client/scss/components/_listing.scss#L391-L438) are unused and can be removed

Code search regex for index styles: `["'][^"']+[^_:/a-z.]index[^_:/[a-z.][^"']+["']`

CSS coverage report from Chrome:

chrome-css-coverage

### PR DESCRIPTION
**Pull request by PaarthAgarwal**
Fixes #8867.
Moved _form-side.scss from layout to components.
Deleted some now redundant .index styles.

### CODE REVIEW

**Comment by lb-** [comment]
When you get a min - please update the PR title to be a bit clearer.

File: client/scss/components/_listing.scss [hunk 1]
```diff
@@ -388,67 +388,6 @@ ul.listing {
...
-        &:active {
-          color: $color-white;
-          background: #484848;
-          border-color: #333;
-        }
-      }
-    }
-  }
-
-  .indicator {
```
Comment (lb-) [inline client/scss/components/_listing.scss:440]: A heads up - I we need to keep the indicator and closing bracket underneath.

I have fixed this up and will push a new update though.

**Review by lb- — APPROVED** [review]
This was super close @PaarthAgarwal  I just ran the checks that Thibaud recommended and did a few smoke tests and was able to also remove the other `.index` styles.

Will merge in shortly.

**Comment by PaarthAgarwal** [comment]
> This was super close @PaarthAgarwal I just ran the checks that Thibaud recommended and did a few smoke tests and was able to also remove the other `.index` styles.
> 
> Will merge in shortly.

Ahh it was work in progress. I just did some changes pushed them to cloud and thought I'll do the rest later from phone. Great to know it was close haha

**Pull request merged**